### ISSUE TITLE
Spanish model using rasa (spaCy)

### ISSUE THREAD

**Issue by parvathysarat** [RasaHQ/rasa_nlu#671]
I am using Rasa NLU to train a Spanish model. I have the es_core_web_sm (as well as English) models of spaCy installed. My issue is that I am not sure if the spacy Spanish language model has been loaded or if only the custom-trained texts/entities/values are being added to a blank model, because I'm not getting intents with good confidence levels for basic spanish sentences.

My config_spacynew.json file : 
`{
  "language" : "es",
  "spacy_model_name" : "es",
  "pipeline": "spacy_sklearn",
  "path" : "./projects",
  "data" : "./stuff.json"
}
`

Should I be making other changes/ elsewhere?
I'm a beginner at this, any help is appreciated. Thanks!
Code: 
```
import spacy
import requests
import json
import sklearn
from rasa_nlu.converters import load_data
from rasa_nlu.config import RasaNLUConfig
from rasa_nlu.model import Trainer

training_data = load_data('stuff.json')
trainer = Trainer(RasaNLUConfig("config_spacynew.json"))
trainer.train(training_data)
model_directory = trainer.persist('./projects/default/')
from rasa_nlu.model import Metadata, Interpreter

# where `model_directory points to the folder the model is persisted in
interpreter = Interpreter.load(model_directory, RasaNLUConfig("stuff.json"))
print interpreter.parse(u"como estas")

Fitting 2 folds for each of 6 candidates, totalling 12 fits
[Parallel(n_jobs=1)]: Done  12 out of  12 | elapsed:    0.1s finished
{u'entities': [], u'intent': {u'confidence': 0.51586523913578286, u'name': u'greet'}, 'text': u'como estas', u'intent_ranking': [{u'confidence': 0.51586523913578286, u'name': u'greet'}, {u'confidence': 0.19394466990205211, u'name': u'hazard reporting'}, {u'confidence': 0.11179921846426252, u'name': u'affirm'}, {u'confidence': 0.061330332434286455, u'name': u'hazard logging'}, {u'confidence': 0.059356678259903005, u'name': u'safety recommendation'}, {u'confidence': 0.025392835701401759, u'name': u'situational awareness'}, {u'confidence': 0.017844732793248858, u'name': u'restaurant_search'}, {u'confidence': 0.014466293309062768, u'name': u'goodbye'}]}
[Finished in 17.3s]
```
 

**Rasa **Content of configuration file** (if used & relevant):NLU version ** : 0.10.3

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...): spacy_sklearn Python 2.7

**Operating system** (windows, osx, ...): Xubuntu

**Comment by wrathagom** [comment]
@parvathysarat how did you download/install the Spanish spacy model? `python -m spacy download es` or `python -m spacy download es_core_web_sm` assuming the latter did you link that model? `python -m spacy link es_core_web_sm es`?

**Comment by parvathysarat** [comment]
I'm not too sure but using `spacy.load('es')`, I'm able to parse Spanish texts and also get certain entities recognized ('Person', 'Location' etc). Also with the model created using Rasa, the metadata file shows the correct Spanish` spacy_model_name` and `language` so I believe I did link the model? So spaCy is just working poorly in this case?
I've switched to MITIE for now, since it works better on smaller datasets. I'd have <500 examples in total, 25-50 examples for each of 6-8 intents.

**Comment by wrathagom** [comment]
It does sound like everything is linked up properly. Unfortunately I would be lost trying to use Rasa with Spanish (really using Spanish at all :/ ).

I've asked the Spanish speaking members of my team if they have anything to add. You may also want to join the Gitter and see if anyone there has used Rasa with Spanish, since I know now everyone in the community watches the issues.

**Comment by parvathysarat** [comment]
Okay, I'll do that. Another clarification I'd like regarding the training data format -
```
{
  "text": "show me chinese restaurants",
  "intent": "restaurant_search",
  "entities": [
    {
      "start": 8,
      "end": 15,
      "value": "chinese",
      "entity": "cuisine"
    }
  ]
}

```
Is it not possible to train a list of entities for one example, separated by commas?

```
{
  "text": "show me chinese restaurants around Lake Street",
  "intent": "restaurant_search",
  "entities": [
    {
      "start": 8,
      "end": 15,
      "value": "chinese",
      "entity": "cuisine"
    },
   {
      #details of entity 2
   } 
   ]
}
```

**Comment by parvathysarat** [comment]
The accuracy improved with MITIE and I'm hoping to better it by adding more synonyms and examples. Thanks for the suggestions and for your time! Btw Chinese support for Rasa NLU using MITIE+jieba has been successful/proved by users, hasn't it? Really looking for multilingual support here.

**Comment by tmbo** [comment]
Yes I think the jieba tokenizer worked quite well. I'll close this for now, let us know if there are any other questions.
